modules/sw360/sw360-workflow/src/test/java/org/eclipse/sw360/antenna/sw360/workflow/generators/SW360UpdaterTest.java

```diff
@@ -77,15 +94,23 @@ public void testConfigure() {
 
         updater.configure(configMap);
 
-        verify(toolConfigMock, times(1)).getProxyHost();
-        verify(toolConfigMock, times(1)).getProxyPort();
+        @SuppressWarnings("unchecked")
+        ArgumentCaptor<SW360ConnectionConfigurationFactory.Getter<String>> captor =
+                ArgumentCaptor.forClass(SW360ConnectionConfigurationFactory.Getter.class);
+        verify(connectionFactory).createConnection(captor.capture(), eq(httpClient), eq(mapper));
+        SW360ConnectionConfigurationFactory.Getter<String> getter = captor.getValue();
+        for (Map.Entry<String, String> e : configMap.entrySet()) {
+            assertThat(getter.apply(e.getKey())).isEqualTo(e.getValue());
+        }
```

Comment:
could this ever assert to false?
and if you are testing the getter.apply method, is this the right place?

Reply:
see my other comment